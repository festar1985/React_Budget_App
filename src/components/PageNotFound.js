import React from "react";
import { Link } from "react-router-dom";

const PageNotFound = () => (
  <p>
    404 Page Not found <Link to="/dashboard">Go Home</Link>
  </p>
);

export default PageNotFound;
