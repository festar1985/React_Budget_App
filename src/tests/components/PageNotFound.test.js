import React from "react";
import PageNotFound from "../../components/PageNotFound";
import { shallow } from "enzyme";
import toJSON from "enzyme-to-json";
import Adapter from "enzyme-adapter-react-16";
import Enzyme from "enzyme";

Enzyme.configure({ adapter: new Adapter() });

test("should render Page Not Found", () => {
  const wrapper = shallow(<PageNotFound />);
  expect(toJSON(wrapper)).toMatchSnapshot();
});
