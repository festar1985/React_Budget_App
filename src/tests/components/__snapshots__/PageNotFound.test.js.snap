// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Page Not Found 1`] = `
<p>
  404 Not found 
  <Link
    to="/"
  >
    Go Home
  </Link>
</p>
`;
